src/middlewares/validation.js

```
@@ -2,23 +2,33 @@ const { createError } = require('~/utils/errorsHelper')
+  Object.entries(schema).forEach(([schemaFieldKey, schemaFieldValue]) => {
+    const fullPath = path ? `${path}.${schemaFieldKey}` : schemaFieldKey
+    const reqBodyField = data[schemaFieldKey]
+
+    validateRequired(fullPath, schemaFieldValue?.required, reqBodyField)
+
+    if (reqBodyField) {
+      if (schemaFieldValue.type === 'object' && schemaFieldValue.schema) {
+        validateObject(schemaFieldValue.schema, reqBodyField, fullPath)
+      } else {
+        Object.entries(schemaFieldValue).forEach(([validationType, validationValue]) => {
+          validateFunc[validationType](schemaFieldKey, validationValue, reqBodyField)
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)